deployments/bridges/rialto-millau/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3.5'
+services:
+  relay-millau-headers-to-rialto:
+    image: paritytech/substrate-relay
+    entrypoint: /config/relay-millau-headers-to-rialto-entrypoint.sh
```

Comment:
Same here. I think we can actually have `/entrypoints/relay-headers-millau-rialto.sh`  or `/relay/headers-millau-rialto.sh`